packages/itwinui-css/src/dialog/dialog.scss

```diff
@@ -3,144 +3,105 @@
 @import '../style/index';
 @import '../icon/index';
 
-@mixin iui-dialog-backdrop {
-  @include iui-reset;
-  z-index: 999;
-  isolation: isolate;
-  position: fixed;
+$iui-dialog-width: 380px;
+
+@mixin iui-dialog-wrapper {
+  position: relative;
+  overflow: hidden;
   top: 0;
   left: 0;
   width: 100%;
   height: 100%;
-  @include themed {
-    background-color: rgba(0, 0, 0, t(iui-opacity-4));
-  }
-
-  // Small/Resizable Dialog Animations for CSS
-  visibility: hidden;
-  opacity: 0;
-  transition: visibility $iui-speed-instant linear $iui-speed-fast, opacity $iui-speed-fast ease-out;
-
-  &.iui-dialog-visible {
-    visibility: visible;
-    opacity: 1;
-    // remove delay for entry animation so dialog is instantly visible
-    transition-delay: $iui-speed-instant;
-  }
-}
-
-@mixin iui-dialog-backdrop-relative {
-  position: relative;
-  overflow: hidden;
+  pointer-events: none;
+  transform: translateX(0); // creates containing block for position: fixed
 }
 
 @mixin iui-dialog {
+  z-index: 999;
+  isolation: isolate;
   border-radius: $iui-border-radius;
   box-shadow: $iui-elevation-24;
   box-sizing: border-box;
-  position: absolute;
+  position: fixed;
   padding: $iui-baseline $iui-m;
   box-sizing: border-box;
   padding: $iui-baseline $iui-m;
   overflow: hidden;
-  @include themed {
-    background-color: t(iui-color-background-1);
-  }
+  pointer-events: auto;
+  background-color: var(--iui-color-background-1);
```

Comment:
I don't think we can use plain CSS variables until v1 as it is not a new component.

Reply:
3 reasons I used css var:
- The PR for removing `themed` mixin was already completed in v1 branch, so I don't want to create additional work when we merge branches.
- IE will be officially considered "dead" in 12 days and we might not even release this until then, so there really is absolutely no reason to go through the extra time.
- draggable is new so we can ignore that. In the existing dialog, `background-1` is just white which is the default background for html anyway.

Reply:
But we announced to everyone that we will drop IE with v1 and now we are starting to break IE before that.

Reply:
If it would default to white I would be ok but: 
![image](https://user-images.githubusercontent.com/36186912/172185461-c231eede-369d-498d-9652-1390b520f2c2.png)

Reply:
ok, thanks for checking. I've hardcoded fallback background colors now. Is that fine?